### PR TITLE
Operator multi

### PR DESCRIPTION
**Pull request by mgross**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # MpiIsoApp development version
 
-## Version 23.09.0
+## Version 23.09.1
+
+## New Features
+- [please add some description]
```
Comment (arunge) [inline NEWS.md:6]: Can you please add some description what is new now?

**Review by arunge — CHANGES_REQUESTED** [review]
- I merged the [most recent changes](https://github.com/Pandora-IsoMemo/iso-app/pull/184) from beta: https://github.com/Pandora-IsoMemo/iso-app/pull/186. Can you please test if your new feature is still working as expected?

- Could you please update the news.md?

- Additionally, the checks fail because:
image

Please, remove global variables

File: NEWS.md (hunk 1 above)
Comment (mgross) [reply NEWS.md]: added

**Comment by mgross** [comment]
@arunge still works as expected, I updated the news file and fixed global vars

**Review by arunge — APPROVED** [review]
Looks ok. Checks were passed. Thanks for your updates!

**Comment by arunge** [comment]
@mgross You can merge, or let me know if I should do so! :slightly_smiling_face:

**Pull request merged**